agent/fw_wordpress.c

```diff
@@ -809,7 +810,9 @@ void nr_wordpress_version() {
   if (SUCCESS == result) {
     if (nr_php_is_zval_valid_string(&retval)) {
       char* version = Z_STRVAL(retval);
-      nr_txn_add_php_package(NRPRG(txn), "wordpress", version);
+      nr_txn_add_php_package(NRPRG(txn), PHP_PACKAGE_NAME, version);
```

Comment:
doesn't the add_php_package need to be in an `if vm` block?
Probably would have been easier to pass `vulnerability_management_package_detection_enabled` and have the logic to check for enabled/disabled in the `nr_txn_add_php_package` function to reduce code duplication of everyone checking for it.

Reply:
Fixed: [647d424](https://github.com/newrelic/newrelic-php-agent/pull/868/commits/647d424ffb824368d14f803f6812df43a75be043). I decided to not pass `vulnerability_management_package_detection_enabled` in the `nr_txn_add_php_package` function and leave it like this for now because it requires `php_agent.h` which is in the agent directory and would require `#include "../agent/php_agent.h"`, which is leading to compilation errors.

Reply:
Yes, it's a little tricky getting `agent` info to `axiom`, but in this case, isn't it just a bool that could be passed in?
Then the bool checked in axiom whether to add package or not?

Reply:
It's beyond the scope of this PR to change that, but think about if it could make sense to roll it into another PR.
At a minimum, the description [here](https://github.com/newrelic/newrelic-php-agent/blob/main/axiom/nr_txn.h#L1164) should indicate it should only be called if vm is enabled.